src/main/java/com/ongvida/api/exceptionhandler/ApiExceptionHandler.java

```diff
@@ -0,0 +1,56 @@
+package com.ongvida.api.exceptionhandler;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
```

Comment:
```suggestion
package com.ongvida.api.exceptionhandler;

import java.time.LocalDateTime;
import java.util.ArrayList;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.context.MessageSource;
import org.springframework.context.i18n.LocaleContextHolder;
```

remove essa linha em branco e deixa os imports com um espaço depois do package, segue essa sugestão pro projeto todo